src/containers/search_result.jsx

```diff
@@ -130,6 +147,29 @@ class SearchResult extends Component {
             );
           })}
         </InnerDiv>
+
+        <InnerSortOperationDiv>
+          <Span>Sort by</Span>
+          <SelectField
+            value={query.sortMethodValue}
+            onChange={(e, index, value) => { this.onSortMethodChange(value); }}
+          >
+            {
+              [
+                { value: 'relevance', primaryText: 'Relevance' },
```

Comment:
You can extract a constant outside the component to store this configuration. It will make the jsx code clear. For example:
```
const sortMethods = [
    {....},
    ....
]
```
Then, at the jsx, you can simply use:
```
sortMethods.map((props) => (
   <MenuItem {...props} />
))
```

Reply:
I just a little bit cannot understand this, this won't be change in today's PR due to the time limitation